File: logic-exercises/eclipse/exe9.java
package eclipse;

import java.util.Scanner;

public class exe9 {
    static public void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um numero: ");
        double a = scanner.nextDouble();
        double fatorial = 1;

        for (int i = 1; i <= a; i++) {
            fatorial *= i;
        }
        System.out.println("O fatorial de " + a + " é: " + fatorial);
        scanner.close();
    }   
}

File: logic-exercises/low-level/exe9.c
#include <stdio.h>

int main(void) {
    int a;
    double fatorial = 1;

    printf("Digite um numero: ");
    scanf("%d", &a);

    for (int i = 1; i <= a; i++) {
        fatorial *= i;
    }
    printf("O fatorial de %d e: %.0f", a, fatorial);

    return 0;
}